static/less/ureport2.less

```diff
@@ -89,6 +89,56 @@ body {
   }
 }
 
+#ureport-policies-bar{
```

Comment:
Some cases have a space between the id and the `{` and others don't. What's the standard?

Reply:
Done. U-Report doesn't have a standard.